assets/helpers/abtest.js

```diff
@@ -20,7 +20,7 @@ type Audience = {
   size: number,
 };
 
-type TestId = 'noTestDefined';
+type TestId = 'noTestDefined' | 'contributionsLandingAddingMonthly';
```

Comment:
Are we using the `noTestDefined` type? maybe we can remove since we have a test now.